project/webapp/templates/thread.html

```diff
@@ -6,6 +6,24 @@
     {% include "base/less_headers/thread_less.html" %}
 </head>
 
+<script>
+$(document).ready(function() {
```

Comment:
This script should be unnecessary, if you simply change the backend logic to not check for a POST request.